src/test/java/org/jabref/model/database/BibDatabaseContextTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jabref.model.database;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.model.metadata.FileDirectoryPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertTrue;
+
+public class BibDatabaseContextTest {
+
+    private String currentWorkingDir;
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    FileDirectoryPreferences preferences;
+
+    @Before
+    public void setUp() {
+        Map<String, String> mapFieldDirs = new HashMap<>();
+        preferences = new FileDirectoryPreferences("saulius", mapFieldDirs, true);
+        currentWorkingDir = Paths.get(System.getProperty("user.dir")).toString();
```

Comment:
I think it is preferable to use [temporary folder rule of junit](http://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html) here, so that the test files are automatically deleted afterwards.